stablehlo/transforms/VhloToVersion.cpp

```diff
@@ -326,6 +348,8 @@ void populateVhloToVersionPatterns(RewritePatternSet* patterns,
   patterns->add<vhlo::CollectivePermuteOpV2ToV1>(*converter, context);
   patterns->add<vhlo::AllGatherOpV1ToV2>(*converter, context);
   patterns->add<vhlo::AllGatherOpV2ToV1>(*converter, context);
+  patterns->add<vhlo::AllToAllOpV1ToV2>(*converter, context);
```

Comment:
Doesn't have to be here, but I think we can add multiple patterns in a single invocation. Would help pair the upgrade/downgrade:

```
patterns->add<vhlo::AllToAllOpV1ToV2, vhlo::AllToAllOpV2ToV1>(*converter, context);
```

Reply:
I've added this to my to-do list. Just wanted to make note while reviewing.